runtime/j9vm/jvm.c

```diff
@@ -3577,69 +3577,25 @@ JVM_LoadSystemLibrary(const char *libName)
  *	DLL: jvm
  */
 
-/* NOTE THIS IS NOT REQUIRED FOR 1.4 */
+/* NOTE THIS IS NOT REQUIRED FOR 1.4
+ * Additional note: this is required by JDK15+ jdk.internal.loader.NativeLibraries.load().
+ *  it is only invoked by jdk.internal.loader.BootLoader.loadLibrary(),
+ */
 
 void* JNICALL
-JVM_LoadLibrary(const char *libName)
+JVM_LoadLibrary(const char* libName)
 {
-	JNIEnv *env;
-	JavaVM *vm = (JavaVM *) BFUjavaVM;
-	char errMsg[512];
+	void* result = NULL;
+	J9NativeLibrary* nativeLibrary = NULL;
+	J9JavaVM* javaVM = (J9JavaVM*)BFUjavaVM;
+	J9InternalVMFunctions* vmFuncs = javaVM->internalVMFunctions;
+	J9VMThread* currentThread = vmFuncs->currentVMThread(javaVM);
 
-#ifdef WIN32
-	HINSTANCE dllHandle;
-	UINT prevMode;
-
-	Trc_SC_LoadLibrary_Entry(libName);
```

Comment:
The tracepoints should remain.

Reply:
and the tracepoint levels can be modified to level 2

Reply:
Fixed.